moved/src/move_execution/eth_token.rs

```diff
@@ -198,7 +199,6 @@ pub fn get_eth_balance<G: GasMeter>(
 }
 
 /// Simplified API for getting the base token balance for use in tests.
```

Comment:
We should update the comment now that this is used outside of tests. Include a warning that this API should not be used in any transaction processing; it should only be used when a VM session does not already exist as part of the request pipeline.